custom/icds_reports/templates/icds_reports/mobile_login.html

```diff
@@ -1,12 +1,71 @@
-{% extends "icds/login.html" %}
+{% extends "icds_reports/base_minimal.html" %}
 {% load i18n %}
-{% block title %}{% trans "Login to ICDS CAS Mobile Dashboard" %}{% endblock title %}
-{% block page_content %}
-{{ block.super }}
+{% load staticfiles %}
+{% load hq_shared_tags %}
+{% block title %}{% trans "Login to ICDS CAS Mobile Dashboard" %}{% endblock %}
+{% block head %}
+  <link type="text/css" rel="stylesheet" media="all" href="{% static 'bootstrap/dist/css/bootstrap.css' %}"/>
+  <link type="text/css" rel="stylesheet" media="all" href="{% static 'icds_reports/mobile/css/app.css' %}"/>
+  <script src="{% static 'bootstrap/dist/js/bootstrap.js' %}"></script>
```

Comment:
You can remove this now, it gets included by the `javascript_libraries` in the parent.